src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,7 +1,13 @@
+<button mat-icon-button aria-label="Delete task" color="primary">
+  Clear All Done Tasks
+  <mat-icon aria-label="Delete task" (click)="clearAllDoneTasks()">delete</mat-icon>
```

Comment:
it is a bit unexpected just to have the action on the icon, not the text too

Reply:
I moved it to the button tag, you're right. It's unexpected.